db/migrate/20230117220432_add_foreign_key_to_post.rb

```diff
@@ -0,0 +1,5 @@
+class AddForeignKeyToPost < ActiveRecord::Migration[7.0]
+  def change
+    add_column :posts, :author_id, :integer
+  end
+end
```

Comment:
- [ ] Referntial integrity for the posts table should be done as shown in the screenshot below
![image](https://user-images.githubusercontent.com/81932478/213055662-edaf8353-4a3f-46f1-9033-b27993c551f8.png)
- this will create `author_id` and a corresponding index